cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -194,6 +195,114 @@ describe("Timeline", () => {
+            // .mx_GenericEventListSummary[data-layout="irc"] > .mx_EventTile_line
+            //  = var(--name-width) + var(--icon-width) + $MessageTimestamp_width + 2 * var(--right-padding)
+            //  = 80 + 14 + 46 + 2 * 5
+            //  = 150px
+            cy.get(".mx_GenericEventListSummary[data-layout=irc] > .mx_EventTile_line").should(
+                "have.css",
+                "padding-inline-start",
+                "150px",
+            );
+            // Check width and spacing values of elements in .mx_EventTile, which should be equal to 150px
+            // --right-padding should be applied
+            cy.get(".mx_EventTile > *").should("have.css", "margin-right", "5px");
+            // --name-width width zero inline end margin should be applied
+            cy.get(".mx_EventTile .mx_DisambiguatedProfile")
+                .should("have.css", "width", "80px")
+                .should("have.css", "margin-inline-end", "0px");
+            // --icon-width should be applied
+            cy.get(".mx_EventTile .mx_EventTile_avatar > .mx_BaseAvatar").should("have.css", "width", "14px");
+            // $MessageTimestamp_width should be applied
+            cy.get(".mx_EventTile > a").should("have.css", "min-width", "46px");
+            // Record alignment of collapsed GELS and messages on messagePanel
+            cy.get(".mx_MainSplit").percySnapshotElement("Collapsed GELS and messages on IRC layout", { percyCSS });
+
+            // 2. Alignment of expanded GELS and messages
+            // Click "expand" link button
+            cy.get(".mx_GenericEventListSummary_toggle[aria-expanded=false]").click();
+            // Check inline start spacing of info line on expanded GELS
+            cy.get(".mx_EventTile[data-layout=irc].mx_EventTile_info:first-of-type .mx_EventTile_line")
+                // See: _EventTile.pcss
+                // --EventTile_irc_line_info-margin-inline-start
+                // = 80 + 14 + 1 * 5
+                .should("have.css", "margin-inline-start", "99px");
+            // Record alignment of expanded GELS and messages on messagePanel
+            cy.get(".mx_MainSplit").percySnapshotElement("Expanded GELS and messages on IRC layout", { percyCSS });
+
+            // 3. Alignment of expanded GELS and placeholder of deleted message
+            // Delete the second (last) message
+            cy.get(".mx_RoomView_MessageList > .mx_EventTile_last").realHover();
+            cy.get(".mx_RoomView_MessageList > .mx_EventTile_last .mx_MessageActionBar_optionsButton", {
+                timeout: 1000,
+            })
+                .should("exist")
+                .realHover()
+                .click({ force: false });
+            cy.get(".mx_IconizedContextMenu_item[aria-label=Remove]").should("be.visible").click({ force: false });
+            // Confirm deletion
+            cy.get(".mx_Dialog_buttons button[data-testid=dialog-primary-button]")
+                .should("be.visible")
```

Comment:
`.should("have.text", "Remove")` should be used instead in case the class name of the confirmation button for removal is switched with that of cancel button.

Reply:
Done -> https://github.com/matrix-org/matrix-react-sdk/pull/10197/commits/4ec04d546cebe56905abaddd7b3328dfaa1a6534